assets/csidriveroperators/azure-disk/08_deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: azure-disk-csi-driver-operator
+  namespace: openshift-cluster-csi-drivers
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: azure-disk-csi-driver-operator
+  strategy: {}
+  template:
+    metadata:
+      labels:
```

Comment:
Can you please add the annotation:
```
annotations:
        workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```
it is critical for our work in support of the workload partitioning feature
(openshift/enhancements#703)